src/libraries/System.Text.Json/ref/System.Text.Json.csproj

```diff
@@ -17,6 +17,10 @@
     <Reference Include="System.Threading.Tasks.Extensions" />
     <Reference Include="Microsoft.Bcl.AsyncInterfaces" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)'">
```

Comment:
This itemgroup is wrong, you should never compile implementation code (specially from corelib) via a reference project. Instead you should be referencing the contract where we are exposing DynamicallyAccessedMembersAttribute to get that reference

Reply:
For the other frameworks where this type doesn't exist, you should instead just #ifdef or separate that into a different file that gets conditionally includded only for netcoreappcurrent.

Reply:
Fixed, thanks a lot!